teacher.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'person'
+
+class Teacher < Person
+  attr_accessor :specialization
+
+  def initialize(age, parent_permission: true, name: 'Unknown')
```

Comment:
Same here, Kindly include `specialization` as an argument of the `initialize` method, otherwise the value of `specialization` in `@specialization = specialization` will always be undefined.

Reply:
Hi @KDlamini,
Thank you so much for bringing my attention to this offense, I have included the `specialization` as an argument of the `initialize` method. 👍